src/extensions/agent_modules/shared/state_store/tests/agent_state_store_ut.cpp

```diff
@@ -20,14 +20,14 @@ TEST_CASE("communication handle")
 {
     SECTION("Valid handle")
     {
-        ADUC_STATE_STORE_RESULT result = ADUC_StateStore_Initialize(TEST_STATE_FILE);
+        ADUC_STATE_STORE_RESULT result = ADUC_StateStore_Initialize(TEST_STATE_FILE, false /* isUsingProvisioningService */);
```

Comment:
May not be apart of this PR but we should start being more aggressive about happy + unhappy paths making sure we're exiting the way we want.